src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1325,7 +1372,19 @@ void Transformations::Snippets(void) {
     }
 
     CPU_DEBUG_CAP_TRANSFORMATION_SCOPE(this, Snippets);
+// Disable MainSnippets for int8 models on arm platforms
```

Comment:
1. Can we move this check into the beggining of MainSnippets?
2. Lets mentione that this is due to perf reason
3. Lets create ticket on SNippets part and put the number here

Reply:
Fixed